gradle/spotless.gradle

```diff
@@ -2,16 +2,36 @@ apply plugin: 'com.diffplug.gradle.spotless'
 
 spotless {
   java {
+    googleJavaFormat()
     licenseHeaderFile rootProject.file('gradle/enforcement/spotless.license.java'), '(package|import|public)'
     target 'src/**/*.java'
   }
   groovy {
     licenseHeaderFile rootProject.file('gradle/enforcement/spotless.license.java'), '(package|import|class)'
   }
   scala {
+    scalafmt()
     licenseHeaderFile rootProject.file('gradle/enforcement/spotless.license.java'), '(package|import|public)'
   }
   kotlin {
+    // ktfmt() // Requires newer version of java
```

Comment:
which one?

Reply:
is requires Java 11, which is fine for us, so i tried it, but it only supports 4-space indent, i updated comment